.storybook/webpack.config.js

```diff
@@ -1,10 +1,12 @@
 const path = require('path');
 const SvgStorePlugin = require('external-svg-sprite-loader');
 
-module.exports = (config) => {
+module.exports = ( config ) => {
```

Comment:
What's the reason for this change?

Reply:
None. They're remnants of trying to upgrade storybook's version to v5. Changing it back to what it was.